src/pull-issues-and-create-database.ts

```diff
@@ -0,0 +1,20 @@
+import getIssuesByProject from './api-functions/get-issues-by-project'
+import BQ from './big-query-functions/big-query-service-class';
+import issueSchema from './schemas/issue-schema'
+
+export default async function pullIssuesAndCreateDatabase(datasetId: string, projectId: string) {
+    const issues = await getIssuesByProject(projectId);
```

Comment:
Service class should not be responsible to do external operations other than big query-related things. so you should not be mixing them here. You should follow the Single responsibility principle.